### DISCUSSION TITLE
如何实现点击节点之后把所有以此节点为终点的边高亮

### DISCUSSION THREAD

**Discussion by Oct16th-Lcs** [didi/LogicFlow#1224] [Q&A] [answered]
按照这个回答试了一下好像不行
![image](https://github.com/didi/LogicFlow/assets/46834744/225c0b6d-8e25-49ce-9a72-cd77fd03aac7)

**Comment by towersxu** [comment, marked as answer]
这个答案是旧版本的，在1.0版本之后，需要通过自定义连线和节点来实现。
https://github.com/didi/LogicFlow/issues/195#issuecomment-930100966

**Reply by Oct16th-Lcs** [reply]
可以了，感谢
